contents/docs/feature-flags/common-questions.mdx

```diff
@@ -92,6 +97,49 @@ A workaround is to create a dynamic behavioral cohort and then duplicate it as a
     classes="rounded"
 />
 
+Another option is to set a person property on the event and then target that property. For example, you can set a person property on a `signed up` event and then target the `business_industry` property with the feature flag.
```

Comment:
Are they "person properties on events" or is just "event properties?" Not rhetorical, actually asking!

Reply:
We differentiate between `person properties` and `group properties`, so in this case I think he's trying to make it clear that we will enrich these events with person properties.

Reply:
ah ha, thank you!